src/screens/NewTopicScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { postNewTopic}from '../api/DataHandler';
+
+//import { Link } from 'react-router-dom'
+
+interface NewTopicState{
+    post_title: string,
+    post_text: string,
+    post_url: string,
+    username: string,
+    pwdHash: string,
+    hanesst_id: number,
+}
+class NewTopicScreen extends React.Component<any, NewTopicState> {
+
+state = {
+    post_title: "",
+    post_text: "",
+    post_url: "",
+    username: "",
+    pwdHash: "",
```

Comment:
Og så give dem en start værdi på undefined.

Reply:
Caused me some issues with using "undefined", mainly due to me totally forgetting to add //@ts-ignore above the places where it was used.

Has been fixed

Reply:
Now I also remember why I did as I did.

While the function still executes and works, the following warning is shown in the Chrome-Console:

`Warning: A component is changing an uncontrolled input of type text to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
    in input (created by PostScreen)
    in label (created by PostScreen)
    in form (created by PostScreen)
    in div (created by PostScreen)
    in div (created by PostScreen)
    in PostScreen (created by Route)
    in Route (created by App)
    in div (created by App)
    in Router (created by BrowserRouter)
    in BrowserRouter (created by App)
    in App`